src/app/pages/collections/collections.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { Collection } from 'src/app/models/Collection';
+import { Link } from 'src/app/models/Link';
+
+@Component({
+  selector: 'app-collections',
+  templateUrl: './collections.component.html',
+  styleUrls: ['./collections.component.scss'],
+})
+export class CollectionsComponent {
+  collections: Collection[] = [];
+  collection!: Collection;
+  title = new FormControl('');
```

Comment:
For future reference, can be refactored as a form group with formBuilder

Reply:
Will be done in Sprint 2 !